aspnet/fundamentals/static-files.rst

```diff
@@ -13,29 +13,32 @@ Static files, which include HTML files, CSS files, image files, and JavaScript f
 Serving static files
 --------------------
 
-By default, static files are stored in the `webroot` of your project. The location of the webroot is defined in the project's ``hosting.json`` file where the default is `wwwroot`.
+Static files are typically located in the web root (*<content-root>/wwwroot*) folder. See Content root and Web root in  :doc:`/conceptual-overview/aspnet` for more information. You generally set the content root to be the current directory so that your project's webroot will be found while in development.
 
-.. code-block:: none 
+.. literalinclude:: ../../common/samples/WebApplication1/src/WebApplication1/Program.cs
+  :language: c#
+  :lines: 12-22
+  :emphasize-lines: 5
+  :dedent: 8
 
-  "webroot": "wwwroot"
 
-Static files can be stored in any folder under the webroot and accessed with a relative path to that root. For example, when you create a default Web application project using Visual Studio, there are several folders created within the webroot folder - ``css``, ``images``, and ``js``. In order to directly access an image in the ``images`` subfolder, the URL would look like the following:
+Static files can be stored in any folder under the *wwwroot* and accessed with a relative path to that root. For example, when you create a default Web application project using Visual Studio, there are several folders created within the *wwwroot*  folder - *css*, *images*, and *js*. The URI to access an image in the *images* subfolder:
 
-  \http://<yourApp>/images/<imageFileName>
+- \http://<app>/images/<imageFileName>
+- \http://localhost:9189/images/banner3.svg
 
-In order for static files to be served, you must configure the :doc:`middleware` to add static files to the pipeline. This specific middleware can be configured by adding a dependency on the Microsoft.AspNetCore.StaticFiles package to your project and then calling the ``UseStaticFiles`` extension method from ``Startup.Configure`` as follows:
 
-.. code-block:: c#
+In order for static files to be served, you must configure the :doc:`middleware` to add static files to the pipeline. The static file middleware can be configured by adding a dependency on the *Microsoft.AspNetCore.StaticFiles* package to your project and then calling the :dn:method:`~Microsoft.AspNetCore.Builder.StaticFileExtensions.UseStaticFiles` extension method from ``Startup.Configure``:
```

Comment:
How does `~Microsoft.AspNetCore.Builder.StaticFileExtensions.UseStaticFiles` render in the docs? Since it's an extension method it would be more recognizable as `IApplicationBuilder.UseStaticFiles`.